src/pages/home-page/HomePage.jsx

```diff
@@ -0,0 +1,151 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import "./home-page.css";
+import {Category,Header,useDataFromServer} from "../../components";
+import {tshirt} from '../../assets/images';
+
+const HomePage = () => {
+  const {state,dispatch} = useDataFromServer();
+
+  useEffect(async () => {
```

Comment:
Never make **useEffect callback async***.  To call async functions from useEffect, do it inside the normal callback function.

Read this for more info : https://stackoverflow.com/questions/53332321/react-hook-warnings-for-async-function-in-useeffect-useeffect-function-must-ret

Example :

``` 
   useEffect(()=>{ // Not an async function
      async function makeNetworkCall(){
       /// Make axios or fetch calls
       }
       makeNetworkCall(); // Call your async function here
    },[])
```

Reply:
Thanks for feedback. Implemented it.